### ISSUE TITLE
Сменить тип возвращаемого значения у лямбд в ScheduleManager

### ISSUE THREAD

**Issue by Erdenian** [Erdenian/StudentAssistant#29]
**[Original report](https://bitbucket.org/Erdenian/studentassistant/issue/29) by Ilya Solovyev (Bitbucket: [Erdenian](https://bitbucket.org/Erdenian), GitHub: [Erdenian](https://github.com/Erdenian)).**

----------------------------------------

Пример:

```
#!kotlin

fun <T> getSemesters(block: (Semester) -> T) = readableDatabase.use {
      it.query(Tables.SEMESTERS, null, null, null, null, null, Tables.Semesters.FIRST_DAY).use {
        if (it.moveToFirst()) {
          val indexes = it.semesterColumnsIndexes
          do block(it.createSemester(indexes))
          while (it.moveToNext())
        }
      }
    }
```
Должно быть:

```
#!kotlin

fun getSemesters(block: (Semester) -> Unit) = readableDatabase.use {
      it.query(Tables.SEMESTERS, null, null, null, null, null, Tables.Semesters.FIRST_DAY).use {
        if (it.moveToFirst()) {
          val indexes = it.semesterColumnsIndexes
          do block(it.createSemester(indexes))
          while (it.moveToNext())
        }
      }
    }
```

**Comment by Erdenian** [comment]
**Original comment by Ilya Solovyev (Bitbucket: [Erdenian](https://bitbucket.org/Erdenian), GitHub: [Erdenian](https://github.com/Erdenian)).**

----------------------------------------

Resolve #29

**Comment by Erdenian** [comment]
**Original comment by Ilya Solovyev (Bitbucket: [Erdenian](https://bitbucket.org/Erdenian), GitHub: [Erdenian](https://github.com/Erdenian)).**

----------------------------------------

Merged in erdenian/issue#29 (pull request #33)

Resolve #29
